README.md

```
@@ -85,6 +85,55 @@ This microservice is built on Java, Spring boot framework and MySQL DB.
+     npm --version
+     ```
+
+2. **Install dependencies**
+   - From the project root directory, run:
+     ```
+     npm ci
+     ```
+   - This will install all required dependencies including Husky and commitlint
+
+3. **Verify hooks installation**
+   - The hooks should be automatically installed by Husky
```

markdownlint-cli2 (0.17.2)

Unordered list style
Expected: asterisk; Actual: dash

(MD004, ul-style)